gobblin-yarn/src/main/java/org/apache/gobblin/yarn/YarnService.java

```diff
@@ -479,32 +495,48 @@ public synchronized void requestTargetNumberOfContainers(int numTargetContainers
 
       this.eventBus.post(new ContainerReleaseRequest(containersToRelease));
     }
-
+    this.yarnContainerRequest = yarnContainerRequestBundle;
```

Comment:
Where is this member field being accessed? Do we need it?

Reply:
It is being used by the private class AMRMClientCallbackHandler for determining the helix tag that need to be assigned to the allocated container